reference/employer/openapi.yaml

```diff
@@ -0,0 +1,1083 @@
+                        - 54719e14-5ea7-4fda-9898-f4d9ccb83c1a
+                        - 0ab620fc-203e-44ad-916c-facc69250f6f
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                      description: The unique id for the payment.
+                      pattern: '[0-9a-f]{8}[-]?(?:[0-9a-f]{4}[-]?){3}[0-9a-f]{12}'
+                    pay_period:
+                      type: object
+                      description: The pay period object.
+                      nullable: true
+                      properties:
+                        start_date:
+                          $ref: '#/components/schemas/Date'
+                        end_date:
+                          $ref: '#/components/schemas/Date'
+                    pay_date:
+                      $ref: '#/components/schemas/Date'
+                    debit_date:
+                      $ref: '#/components/schemas/Date'
+                    company_debit:
+                      $ref: '#/components/schemas/Money'
+                    gross_pay:
+                      $ref: '#/components/schemas/Money'
+                    net_pay:
+                      $ref: '#/components/schemas/Money'
+                    employer_taxes:
+                      $ref: '#/components/schemas/Money'
+                    employee_taxes:
+                      $ref: '#/components/schemas/Money'
+                    individual_ids:
+                      type: array
+                      description: Array of every individual on this payment.
+                      nullable: true
+                      items:
+                        type: string
+                        pattern: '[0-9a-f]{8}[-]?(?:[0-9a-f]{4}[-]?){3}[0-9a-f]{12}'
+      summary: Payment
+      operationId: get-payment
+      tags:
+        - Employer
+  /employer/pay-statement:
+    post:
+      description: Read detailed pay statements for each individual.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                requests:
+                  type: array
+                  description: The array of batch requests.
+                  items:
+                    type: object
+                    properties:
+                      payment_id:
+                        type: string
+                        description: A stable Finch `id` (UUID v4) for a payment.
+                        pattern: '[0-9a-f]{8}[-]?(?:[0-9a-f]{4}[-]?){3}[0-9a-f]{12}'
+                      limit:
+                        type: integer
+                        description: Number of pay statements to return (defaults to all).
+                      offset:
+                        type: integer
+                        description: Index to start from.
+                        default: 0
```

Comment:
Since this is not recommended, thoughts on just not documenting this? Also the Try It defaults limit to 0. May be weird/confusing 🤷🏾